package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "issue-finder",
-  "version": "2.2.0",
+  "version": "3.0.0",
```

Comment:
The Version get's also bumped automatically 👍

Reply:
True. To avoid this, I may have to re-fork the repo with the most recent changes.